gradlewAll

```diff
@@ -17,5 +17,5 @@
 
 # Convenient way to invoke a gradle command with all Scala versions supported
 # by default
```

Comment:
We should probably add a comment here that this is no longer needed, but we are keeping it for backwards compatibility.

Reply:
Maybe we should print deprecated information to users to remind that this `gradlewAll` is unnecessary and will be removed later.